services/src/main/java/com/linkedin/kmf/services/configs/TopicRebalancerConfig.java

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright 2016 LinkedIn Corp. Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+ * file except in compliance with the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.linkedin.kmf.services.configs;
+
+import java.util.Map;
+import org.apache.kafka.common.config.AbstractConfig;
+import org.apache.kafka.common.config.ConfigDef;
+import static org.apache.kafka.common.config.ConfigDef.Range.atLeast;
+
+public class TopicRebalancerConfig extends AbstractConfig {
+
+  private static final ConfigDef CONFIG;
+
+  public static final String ZOOKEEPER_CONNECT_CONFIG = CommonServiceConfig.ZOOKEEPER_CONNECT_CONFIG;
+  public static final String ZOOKEEPER_CONNECT_DOC = CommonServiceConfig.ZOOKEEPER_CONNECT_DOC;
+
+  public static final String TOPIC_CONFIG = CommonServiceConfig.TOPIC_CONFIG;
+  public static final String TOPIC_DOC = CommonServiceConfig.TOPIC_DOC;
+
+  public static final String REBALANCE_EXPECTED_RATIO_CONFIG = "rebalance.expectedPartitionBrokerRatio";
+  public static final String REBALANCE_EXPECTED_RATIO_DOC = "The expected ratio of partition / broker.  When the actual ratio falls below this new partitions are created.";
+
+  public static final String REBALANCE_INTERVAL_MS_CONFIG = "rebalance.intervalMs";
+  public static final String REBALANCE_INTERVAL_MS_DOC = "The gap in ms between the times the cluster balance on the monitored topic is checked.";
+
+  public static final String PARTITIONS_PER_BROKER_CONFIG = ProduceServiceConfig.PARTITIONS_PER_BROKER_CONFIG;
+  public static final String PARTITIONS_PER_BROKER_DOC = ProduceServiceConfig.PARTITIONS_PER_BROKER_DOC;
+
+  public static final String TOPIC_REPLICATION_FACTOR_CONFIG = ProduceServiceConfig.TOPIC_REPLICATION_FACTOR_CONFIG;
```

Comment:
I think the job of rebalance service is to increase partition number or reassign partitions. The decision of replication factor should stay with whoever create the topic. Therefore this service probably use the existing replication factor.